lib/view.js

```diff
@@ -37,7 +37,10 @@ export function extractViews(entities) {
 
   Object.keys(entities).forEach((entityId) => {
     const entity = entities[entityId];
-    if (entity.attributes.view) {
+    // Entity is a view if it has the 'view' attribute set.
+    // Consider default_view as view by default. 
+    if (entity.attributes.view ||
+        (entity.entity_id === DEFAULT_VIEW_ENTITY_ID && entity.attributes.view !== false)) {
```

Comment:
You test `entity.attributes.view !== false`, which is equivalent to testing `entity.attributes.view` which is what we do in the first part of this logical expression.